### PR TITLE
Update to adjust expando column width to fit text on one line.

### PR DESCRIPTION
**Pull request by andrethomas6**
Update to adjust expando column width to fit text on one line.

### CODE REVIEW

**Review by sanjuroj — APPROVED** [review]
Not that you're changing anything related to accessibility, but since you're here, did you check to see if the expando is keyboard accessible?  Do we need to handle Enter commands as well?

**Comment by andrethomas6** [comment]
Hey @sanjuroj I added aria labels, tab and escape key functionality for the IPTT accordion. I also added the ability to escape out of three modals with the escape key.

File: scss/_iptt_table.scss [hunk 1]
```diff
@@ -126,10 +126,13 @@ td.iptt-level-row, td.ip-level-row {
 }
 
 td.expando-cell {
-    min-width: 125px;
     color: $blue;
     cursor: pointer;
     border-right: none;
+    div { // Will adjust the width be long enouugh to fit text on one line.
```
Comment (sanjuroj) [inline scss/_iptt_table.scss:132]: typo on "enouugh"

**Review by sanjuroj — APPROVED** [review]
Looks good, thanks for making the updates!

**Pull request merged**